src/SourceGenerators/CommonCode/CommonCode.csproj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <RootNamespace />
```

Comment:
I'd rename the folder to `Generators` (the entire project is about *source* generators anyway, and I like shorter names). Also, instead of `CommonCode`, I'd use `Common` or `Core` instead.

Reply:
Done. I chose `Common` over `Core` because the latter is widely used as a suffix for _dependencies_ containing common types, while here common types are included at a source level.

I have also moved the Common project out of the Generators directory, which simplifies Directory.Build.props as now all projects in that directory are packable. Besides, now Common can use default items, so VS stops adding a single Compile item to the project every time I add a file.